tools/feature_matrix.py

```diff
@@ -106,16 +106,16 @@ def report_missing(setting_name):
     setting_value = getattr(settings, setting_name)
     logger.debug(f'cannot use {feature.name} because {setting_name} is too old: {setting_value}')
 
-  if settings.MIN_CHROME_VERSION < min_versions['chrome']:
+  if min_versions['chrome'] > settings.MIN_CHROME_VERSION:
     report_missing('MIN_CHROME_VERSION')
     return False
-  if settings.MIN_FIREFOX_VERSION < min_versions['firefox']:
+  if min_versions['firefox'] > settings.MIN_FIREFOX_VERSION:
     report_missing('MIN_FIREFOX_VERSION')
     return False
-  if settings.MIN_SAFARI_VERSION < min_versions['safari']:
+  if min_versions['safari'] > settings.MIN_SAFARI_VERSION:
     report_missing('MIN_SAFARI_VERSION')
     return False
-  if 'node' in min_versions and settings.MIN_NODE_VERSION < min_versions['node']:
+  if 'node' in min_versions and min_versions['node'] > settings.MIN_NODE_VERSION:
```

Comment:
Why does the linter want to flip these comparisons?  I think maybe the read more clearly as written.   Maybe revert this file?

Reply:
You must be a fan of Star Wars!

% [`ruff rule SIM300`](https://docs.astral.sh/ruff/rules/yoda-conditions)

Reply:
Maybe its misleading since `settings.MIN_NODE_VERSION` is not so much a constant as a user setting.

In this case we are saying "is the user-specified-min-node-version less then the min node version required for this feature".   So in this `min_versions['node']` (the min node version required for the feature) is really the constant.